chains/evm/contracts/test/ccvs/components/SignatureQuorumValidator/SignatureQuorumValidator.setSignatureConfigs.t.sol

```diff
@@ -62,38 +127,228 @@ contract SignatureQuorumValidator_setSignatureConfigs is SignatureValidatorSetup
     address[] memory signers = new address[](1);
     signers[0] = makeAddr("soloSigner");
     uint8 threshold = 1;
+    uint64 sourceChainSelector = 1;
 
-    s_sigQuorumVerifier.setSignatureConfig(signers, threshold);
+    SignatureQuorumValidator.SignersUpdate[] memory updates = _createUpdate(1, signers, threshold);
+    s_sigQuorumVerifier.applySignersUpdates(new uint64[](0), updates);
 
-    (address[] memory actualSigners, uint8 actualThreshold) = s_sigQuorumVerifier.getSignatureConfig();
+    (address[] memory actualSigners, uint8 actualThreshold) =
+      s_sigQuorumVerifier.getSignatureConfig(sourceChainSelector);
 
     assertEq(actualSigners.length, 1);
     assertEq(actualThreshold, 1);
     assertEq(actualSigners[0], signers[0]);
   }
 
+  function test_applySignersUpdates_RemovalTakesPriorityWithinOneCall() public {
```

Comment:
applySignersUpdates tests should be in a separate file: SignatureQuorumValidator.applySignersUpdates.t.sol

Reply:
Fixed [3c86d94](https://github.com/smartcontractkit/chainlink-ccip/pull/1417/commits/3c86d94c0b26f8f3c93a0af7c5182493e7ca8f91)